File: JXBC-Server/BossComments/apiserver/appbase/models/AccountSignResult.php
<?php

namespace app\appbase\models;

/**
 * @SWG\Definition(required={"SignStatus","CreatedNewPassport"})
 */
class AccountSignResult
{   
    /**
     * @SWG\Property(description="注册(或登录)的结果状态，参见枚举[SignStatus]", type="int")
     */
    public $SignStatus;

    /**
     * @SWG\Property(description="是否创建了新账号", type="boolean")
     */    
    public $CreatedNewPassport;
    
    /**
     * @SWG\Property()
     * @var string
     */
    public $ErrorMessage;

    /**
     * @SWG\Property(description="匿名账号信息", ref="#/definitions/AccountEntity")
     */
    public $Account;
    
    /**
     * @SWG\Property(description="注册(或登录)的后继行为", ref="#/definitions/AdditionalAction")
     */
    public $AdditionalAction;    
      
}

/**
 * @SWG\Definition(required={"ActionType","Source"})
 */
class AdditionalAction
{
    /**
     * @SWG\Property()
     * @var string
     */
    public $ActionType;
    
    /**
     * @SWG\Property(description="来源")
     * @var string
     */
    public $Source;    
}

File: JXBC-Server/appbase/src/JXBC.Web/JXBC.WebCore/ViewModels/AccountSignResult.cs
using System;
using JXBC.Passports;

namespace JXBC.WebCore.ViewModels
{
    /// <summary>
    /// 
    /// </summary>
    public class AccountSignResult
    {
        /// <summary>
        /// 
        /// </summary>
        public SignStatus SignStatus { get; set; }

        /// <summary>
        /// 是否注册新账号
        /// </summary>
        public bool CreatedNewPassport { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public string ErrorMessage { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public AccountEntity Account { get; set; }

        /// <summary>
        /// 后继行为
        /// </summary>
        public AdditionalAction AdditionalAction { get; set; }
    }
}